server/index.ts

```
@@ -0,0 +1,756 @@
+    
+    console.log('📄 No content found, returning empty document');
+    return '<p></p>';
+    
+  } catch (error) {
+    console.error('Error extracting document content:', error);
+    return '<p>Error extracting content</p>';
+  }
+};
+
+// Helper function to convert ProseMirror JSON to HTML
+const convertProseMirrorToHtml = (json: any): string => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)